pkg/ssh/pool.go

```diff
@@ -125,7 +125,7 @@ func getTCPConnection(ctx context.Context, serverAddr string, keepAlive time.Dur
 	return c, nil
 }
 
-func getConn(ctx context.Context, serverAddr string, maxRetries int) (net.Conn, error) {
+func getConn(ctx context.Context, serverAddr string) (net.Conn, error) {
 	var lastErr error
 	t := time.NewTicker(100 * time.Millisecond)
 	for i := 0; i < 5; i++ {
```

Comment:
Shouldn't we use maxRetries here?

Reply:
Agree with @AgustinRamiroDiaz, removing a magic number is always a good solution

Reply:
i've reverted the change by a param `retries` and calling the function with `const defaultRetries = 5` WDYT?